l10n_it_website_sale_corrispettivi/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,14 @@
+**Italiano**
+
+Il comportamento dipende da come è configurato il partner associato all'utente loggato, ad esempio:
+
+* Se il partner associato all'utente loggato ha il flag *Usa corrispettivi* abilitato, la spunta *Voglio fattura* sarà disabilitata;
+* Se il partner associato all'utente loggato ha il flag *Usa corrispettivi* disabilitato, la spunta *Voglio fattura* sarà abilitata;
+
```

Comment:
loggato → che ha fatto/eseguito/effettuato il login

flag/spunta → casella (di selezione/selezionata)

Essendo una casella di selezione:
Usa corrispettivi → Usare corrispettivi
Voglio fattura → Richiedere/Richiesta fattura

Verifica anche in tutte le altre parti del readme.

Reply:
@primes2h per quanto riguarda `Voglio fattura` lo lascerei così in quanto è una campo nel frontend, nella pagina di checkout, dove l'utente sta proporio richiedendo la fattura.

Reply:
> @primes2h per quanto riguarda `Voglio fattura` lo lascerei così in quanto è una campo nel frontend, nella pagina di checkout, dove l'utente sta proporio richiedendo la fattura.

Adesso ho visto.
In casi come questo, dove la casella di selezione non è un'impostazione di configurazione ma corrisponde a un'azione, di solito si usa l'imperativo (come nei pulsanti).

Quindi potrebbe essere "Richiedi fattura".

Se vuoi proprio rivolgerti direttamente all'utente (*) forse è meglio "Richiedo fattura" comunque, "Voglio fattura" mi sembra un po' troppo informale.

Reply:
@primes2h fatto